Description: fix build on 64-bit big-endian systems (PPC64) by adding the
 word member to the payload union
Origin: https://bug627664.bugzilla.mozilla.org/attachment.cgi?id=505742
Bug: https://bugzilla.mozilla.org/show_bug.cgi?id=627664

--- a/js/src/jsval.h	2011-01-20 15:59:49.000000000 +0100
+++ b/js/src/jsval.h	2011-01-20 16:00:21.000000000 +0100
@@ -347,6 +347,7 @@ typedef union jsval_layout
             int32          i32;
             uint32         u32;
             JSWhyMagic     why;
+            jsuword        word;
         } payload;
     } s;
     double asDouble;

